.github/workflows/deployment.yml

```
@@ -49,9 +54,15 @@ jobs:
       run: "yarn install"
 
     - name: "Prepare Website files"
+      if: "inputs.search-index-only == false"
```

`inputs.search-index-only` is guaranteed to be a boolean and therefore does not need to be compared to other booleans, it can be used directly or negated.

```suggestion
      if: "! inputs.search-index-only"
```